CHANGELOG.md

```diff
@@ -19,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added Databricks/Spark setup to the image. See README for setup & usage instructions.
 - Added rollback feature to `Databricks` source.
 - Changed all Prefect logging instances in the `sources` directory to native Python logging.
+- Changed `rm , from_df , to_df` methods in `S3` Source
```

Comment:
This doesn't work, you can always click "Preview" tab and see how the Markdown is rendered. This looks like this: `rm , from_df , to_df`, whereas it should be: `rm()` , `from_df()` , and `to_df()`

Reply:
Updated in:

- https://github.com/dyvenia/viadot/pull/603/commits/9c597385a727078ed1cf9771fee80a17d4139435